MoviesExample/app/src/main/java/com/novoda/demo/movies/MainActivity.java

```diff
@@ -38,16 +38,25 @@ protected void onCreate(Bundle savedInstanceState) {
             public void onMovieSelected(final Movie movie) {
                 startLoadingTrailer(movie);
             }
-
-            @Override
-            public void onPageLoadRequested(final int page) {
```

Comment:
We don't need this callback anymore since the Paging Library decides when to load new data.